addons/math-in-inputs/userscript.js

```diff
@@ -149,9 +141,9 @@ export default async function ({ addon }) {
         mutation.addedNodes.forEach(function (node) {
           if (
             node instanceof HTMLElement &&
-            (node.className.includes("input-group_input-group_plJaJ") ||
-              node.classList.contains("input_input-form_l9eYg") ||
-              node.classList.contains("mediaRecorderPopupContent"))
+            (node.classList.contains(addon.tab.scratchClass("input-group_input-group")) ||
+            node.classList.contains(addon.tab.scratchClass("input_input-form")) ||
```

Comment:
scratchClass isn't supposed to use this way. Use className.includes until the last underscore

Reply:
I have grasped more knowledge of how this addon should work (through experimentation) and plan to fix everything I see wrong with it. Will do tmr (it is 3am)